ga4gh/datamodel/variants.py

```diff
@@ -222,8 +243,13 @@ def generateVariant(self, variantSetId, referenceName, position,
         variant = self._createGaVariant()
         variant.names = []
         variant.referenceName = referenceName
-        variant.id = "{0}:{1}:{2}".format(
-            variant.variantSetId, referenceName, position)
+        md5 = 'notApplicable'
```

Comment:
We should compute the compound ID of the variant in the `AbstractVariantSet` class, and use the values from a GA object to compute the hash. This will allow us to do this in one place. For example, we might have
```
variant.id = self.getVariantId(variant)
```
We can then reuse this for the simulated and the Htslib derived variants.